core/src/Cabana_Version.hpp

```diff
@@ -19,9 +19,9 @@
 namespace Cabana
 {
 
-std::string version();
+std::string version() { return Cabana_VERSION_STRING; }
```

Comment:
These need to be inlined to build downstream

Reply:
Good catch.  Multiple definition error at link time otherwise